src/views/input/MoneyInput.view.js

```diff
@@ -0,0 +1,22 @@
+export default class MoneyInputView {
+  constructor() {
+    this.$target = document.createElement('input');
+    this.$target.setAttribute('type', 'text');
+    this.$target.setAttribute('placeholder', '금액을 입력해주세요');
+    this.render();
+  }
+
+  render() {
+    this.$target.addEventListener('blur', this.blurHandler.bind(this));
+    this.$target.addEventListener('click', this.clickHandler.bind(this));
+  }
+
+  blurHandler() {
```

Comment:
요건 나중에 익히실 네이밍 규칙같은 건데 현재 구현상으로는 함수에 해당하기에

`
```suggestion
 handleBlur() {
```
같은게 어울려보여요

Reply:
오... 아직 모자란 게 너무 많지만, 이런 네이밍 규칙도 조금씩 고려하면서 작성하겠습니다. 감사합니다, 멘토님 :)!